stdlib/public/core/String.swift

```diff
@@ -973,6 +973,23 @@ extension String {
     return _nativeUnicodeUppercaseString(self)
 #endif
   }
+  
+  public // @testable
+  init<T: LosslessStringConvertible>(_ v: T) {
```

Comment:
Please add a space before the colon.  Could you also think of a better parameter name?

```swift
extension String {
  public init<Subject>(_ instance: Subject)
}
```

A doc-comment would also be good to have.